src/App.css

```diff
@@ -1,3 +1,332 @@
 .App-body {
+  background-color: #fafafa;
+}
+
+.navbar {
+  background-color: #fff;
+  display: flex;
+  align-items: baseline;
+  padding-right: 30px;
+  padding-left: 30px;
+}
+
+.links-container {
+  display: flex;
+  column-gap: 20px;
+}
+
+.navbar h1 {
+  width: 15rem;
+  height: 2.313rem;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.faUser {
+  margin-left: auto;
+  color: #0290ff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  background-color: #fafafa;
+}
+
+.links {
+  text-decoration: none;
+}
+
+.BOOKS {
+  width: 3.688rem;
+  height: 1rem;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
```

Comment:
- [ ] Good job so far @adeola003 Your project looks great. However per the project requirements, 

> Your final design should match Zeplin's design (this includes all elements + the same styling)
As a result, there are a few style changes that need to be made to your design :wink: 
- I have attached an image below explaining some of them :+1: 
- Your Design
![image](https://github.com/adeola003/bookstore/assets/102699958/f33178f8-d317-430b-8f92-1e189933ac10)

- The Zeplin Design
![image](https://github.com/adeola003/bookstore/assets/102699958/af6b9d3f-1dba-453e-81be-13f8079a167a)